govuk_test.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "capybara"
-  spec.add_dependency "webdrivers"
+  spec.add_dependency "webdrivers", "~> 4"
```

Comment:
Heads up that this doesn't match what message suggests, this is `>= 4, < 5`. 

It'd be quite reasonable to restrict to `< 5` as who knows what that major version will bring but thought I'd point it out as it didn't seem to match the intention of the documentation.

Reply:
Appreciate there's a bit of nuance there. It seems reasonable to just enforce the minimum restriction unless we know better. Since I also forgot the changelog, here's a PR to make this better https://github.com/alphagov/govuk_test/pull/27